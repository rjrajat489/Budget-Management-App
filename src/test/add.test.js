import {Header} from '../playground/header'
import {shallow} from 'enzyme'
import React from 'react'

test('test component',()=>{
  const x=shallow(<Header/>)
  expect(x).toMatchSnapshot();

})
