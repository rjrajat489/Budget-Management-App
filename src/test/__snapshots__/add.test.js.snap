// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component 1`] = `
<div>
  <h1>
     hi
  </h1>
</div>
`;
